api/src/main/java/io/opentelemetry/api/common/Attributes.java

```diff
@@ -39,47 +51,65 @@
     public AttributesBuilder toBuilder() {
```

Comment:
in order for the javaagent to provide a single implementation that satisfies both the shaded and unshaded interfaces, we need the return value here to be an interface, so that its implementation use a covariant return type that satisfies both, similar to the `TraceState.getTraceState()` example in https://github.com/open-telemetry/opentelemetry-java/issues/1946#issuecomment-727531701

Reply:
done. please take a look